styles/contact.css

```diff
@@ -96,18 +103,65 @@ textarea[type=text-area]:focus {
   bottom: 0;
 }
 
-@media only screen and (min-width: 768px) {
+@media only screen and (min-width: 1090px) {
   form {
     align-items: flex-start;
+    height: 20.625rem;
+  }
+
+  input {
+    height: 1.5rem;
   }
 
   .contact-section {
     display: grid;
-    margin-top: 3.375rem;
-    grid-template-columns: repeat(2, 1fr);
+    height: 30.625rem;
+    margin-top: 1.09375rem;
+    grid-template-columns: repeat(6, 1fr);
+    justify-items: center;
```

Comment:
- [ ] Please use Flexbox to align items in this section. That would be more accurate. Also, please add the background image to match the figma design: 
yours: 
![image](https://user-images.githubusercontent.com/91486791/236193807-188cf901-24d4-4f81-9d0f-89dd865d9608.png)
figma: 
![image](https://user-images.githubusercontent.com/91486791/236193890-625cd6cb-6da8-46e0-a480-5e1a762e0346.png)